parent_topic: "/parent/points_raw"
child_topic: "/child/points_raw"

parent_frame_id: "velodyne"
child_frame_id: "sick"

voxel_size: 1.0

ndt_epsilon: 0.01
ndt_step_size: 0.1
ndt_resolution: 1.0
ndt_iterations: 200

in_x: 0
in_y: 0
in_z: 0
in_roll: 0
in_pitch: 0
in_yaw: 0